api-extractor/src/definitions/ApiModuleVariable.ts

```diff
@@ -0,0 +1,25 @@
+import * as ts from 'typescript';
+import { ApiItemKind, IApiItemOptions } from './ApiItem';
+import ApiMember from './ApiMember';
+
+/**
+ * This class is part of the ApiItem abstract syntax tree. It represents fields of
```

Comment:
>fields [](start = 73, length = 6)

"It represents variables that are exported by an ApiNamespace (or potentially in the future an ApiPackage).  The variables have a name, a type, and an initializer.  The ApiNamespace implementation currently requires them to use a primitive type and be declared as "const"."